src/content/1.7/modules/concepts/controllers/kpi-blocks.md

```diff
@@ -57,3 +57,24 @@ You can follow these steps to easily add a KPI row to a modern page:
         </div>
     {% endblock %}
     ```
+
+## Alter an existing Kpi row
+{{< minver v="1.7.6" title="true" >}}
+
+A hook allows you to alter the list of an existing Kpi row of the Back Office.
+
+This hook is dynamic and is dispatched after the *Kpi row identifier*.
+
+For instance, for a Kpi row identified by "foo":
+
+```php
+// we are in a module
+public function hookActionFooKpiRowModifier(array $params)
+{
+    dump($params['kpis']); // access the complete list
```

Comment:
add a comment with a link to https://symfony.com/doc/current/components/var_dumper.html#the-dump-function it's not a native function and it's specific to a symfony component

Reply:
I prefer relying on ``var_dump()`` function then